charts/generic/k8-custom-metrics-apiserver/engine.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: litmuschaos.io/v1alpha1
```

Comment:
@dharmaanu , this is a custom solution and don't have kubernetes resource associate, we don't need this chaos-chart.